examples/proc_macro_commands.rs

```diff
@@ -0,0 +1,67 @@
+use redis_module::{redis_module, Context, RedisResult, RedisString, RedisValue};
+use redis_module_macros::redis_command;
+
+#[redis_command(
+    {
+        name: "classic_keys",
+        flags: "readonly",
```

Comment:
The think is that Redis API gets those flags as string so I thought it make sense to keep it this way and also be future compatible in case Redis will introduce another such flag.
Notices that on places where Redis uses flags I also used flags.

Reply:
I agree it should have been flags. I just do not like the idea of getting flags here and then construct a string object to pass to Redis. I think it should be the other way around, Redis should change the interface to receive flags and then we should match it.
I guess we can change it to flags now but then each time Redis will introduce a new flag we will require to release a new `redismodule-rs` version and have the module writer to upgrade it.

Reply:
Changed, let me know what you think: https://github.com/RedisLabsModules/redismodule-rs/pull/326/files#diff-50c8282b8ca77e9cc52806fddfd36c9c0462119a7185ae013f7e45d3cfa9f5d2R45

Reply:
https://github.com/RedisLabsModules/redismodule-rs/issues/332